fud2/Cargo.toml

```diff
@@ -33,3 +33,4 @@ path = "src/main.rs"
 [dev-dependencies]
 insta = "1.36.0"
 itertools.workspace = true
+figment = { version = "0.10.12", features = ["toml"] }
```

Comment:
I see why we need this (the main function now needs to touch the config struct), but it's kind of a shame that we need to expose this implementation detail now… not sure there's anything to be done about it, though.

Reply:
That dependency is only used by `fud2` in tests to simulate different config files. I don't think it gets exposed anywhere else.

Reply:
Cool, makes sense! This could go under `[dev-dependencies]` in that case? https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

Reply:
is it not already?

Reply:
ARGH I feel dumb. Sorry!!